site/src/examples/border-layout/BorderItemPosition.module.css

```diff
@@ -0,0 +1,14 @@
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+.container .active {
+  font-weight: var(--salt-text-fontWeight-strong);
+  background: var(--salt-container-secondary-background);
+}
+
+.radioButtonGroup {
+  margin-top: var(--salt-spacing-300);
+  text-transform: capitalize;
```

Comment:
I've noticed repeated use of `text-transform: capitalize` but I'm not sure why? In this case it seems like the labels should just be updated